src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -1547,6 +1547,73 @@ hibernate.connection.url=jdbc:hsqldb:mem:rm;hsqldb.tx=mvcc;hsqldb.lob_file_scale
+recovery feature, if the scheduler crashed at instant T, then the tasks that were on-going would be
+re-scheduled on any node and re-executed from the beginning when the scheduler restarts. With the nodes and task
+recovery feature enabled, when the scheduler restarts it will retrieve the on-going tasks by contacting
+all the resource manager nodes, and the execution of the tasks will continue and finish as if their was no failure of
+the scheduler.
+
+WARNING: The nodes and task recovery feature is *not* applicable when the scheduler is cleanly stopped or exits normally.
+This is a mechanism that is only applicable upon failure of the scheduler (machine crash, abrupt exit).
+
+=== How Does Nodes and Task Recovery Work?
+
+At startup, the scheduler checks the nodes and the tasks that were previously persisted in database to be able to restore
+their state. For each node that is supposed to be handled by the resource manager, if the node is still alive, the
+node's information is restored in the resource manager. The same restoration mechanism is applied for running tasks in
+the scheduler. However at recovery time, if for a node source no node is found in database, then the node recovery will
+not take place for this node source. Instead, the nodes will be redeployed like for the first time.
+On the opposite, if a persisted node is not alive at recovery time, then the node will also be recreated automatically.
+
+A recovery is not a restart, and as such a clean restart of the scheduler will not preserve on-going tasks and
+running nodes. Indeed, in this case the nodes are cleaned and removed from the resource manager before the scheduler
+exits. Thus, there will be no node and no running tasks to recover when the scheduler is started again (only pending
+tasks).
+
+To handle the particular situation in which the scheduler is down when a running task terminates, a proper configuration
+of the scheduler enables the task to hold the result until the scheduler comes back alive.
+
+=== Nodes and Task Recovery Configuration
+
+==== Resource Manager Configuration
+
+In order to enable the nodes and task recovery feature, make sure that the `pa.rm.nodes.recovery` property of the
+resource manager is set to `true`.
+
+==== Scheduler Configuration
+
+In order to force the tasks to keep trying to submit their result to the scheduler until it is up and running, make
+sure that the `pa.scheduler.core.nodepingfrequency` and `pa.scheduler.core.node.ping.attempts` properties of the
+scheduler have values that are big enough.
+
+You can give them a suitable value according to the following description.In the context of the nodes and task recovery,
+`pa.scheduler.core.node.ping.attempts` defines the number of times the task will attempt to submit the task's result to
+the scheduler. `pa.scheduler.core.nodepingfrequency` defines the frequence in seconds at which the task will try to
+contact the scheduler in order to deliver the task's result.
+
```

Comment:
A Node Configuration section is missing, to describe the proactive.node.ping.delay and proactive.node.reconnection.attempts properties